.github/workflows/pr-comment-trigger.yml

```diff
@@ -17,6 +17,9 @@ on:
       UPTEST_CLOUD_CREDENTIALS:
         description: 'Uptest cloud credentials to be passed to the uptest target as environment variable'
         required: true
+      UPTEST_CLOUD_CREDENTIALS_2:
```

Comment:
@ulucinar, I was aware of this problem and it seems that there is no way to pass an arbitrary number of secrets in a reusable workflow. TBH, I was hesitant to solve it in this way, i.e. introducing a new variable for each secret.
I needed something similar in upbound/provider-terraform and used the following option as I think it is cleaner: https://github.com/upbound/provider-terraform/blob/4c847fa292b43d5abf4a0e9203c6ccfeae243944/cluster/test/setup.sh#L9-L27

What do you think?

Reply:
Hi @turkenh,
Agreed. Closing this PR and I'll adapt the provider-aws PR so that we follow the same approach above. Thank you!